### PR TITLE
Update sbt and plugin versions

### PR DESCRIPTION
**Pull request by alsofr**
Those are the changes I had to do in order to build the project locally successfully.

The first errors were related to the fields `ideaPluginName` and `ideaBuild ` which seem to have been renamed.
Then there were errors related to the sbt version.

### CODE REVIEW

**Comment by phiSgr** [comment]
The new versions of IntelliJ plugin stuff is noted. Without seeing your error messages I see no reasons to update them right now. They will be updated when I need to update the extension for Gatling-JavaPB (probably 1 year later).

And please do not include your attempt in supporting streaming in a PR that only says "update versions".

**Comment by alsofr** [comment]
Hi, thanks for the explanation.
Sure thing, let me try again with the latest version.

First attempt is positive, I only had to add the `streamingResponse: Boolean` and continue duplicating the `io.grpc.stub.ClientCalls.asyncUnaryRequestCall` implementation.

I still need to run some tests to make sure it works as expected with regards to:
- response time measurement
- ability to make all responses available in the test.

Is there a way I can push a development version on Sonatype to be able to test the changes on my project without using local jar files?

**Comment by phiSgr** [comment]
I believe I followed [this guide](https://central.sonatype.org/pages/ossrh-guide.html).

**Comment by alsofr** [comment]
Your Intellij changes worked, thank you!
